kvdb/kvdb_etcd.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/lightningnetwork/lnd/kvdb/etcd"
 )
 
-// TestBackend is conditionally set to etcd when the kvdb_etcd build tag is
+// EtcdBackend is conditionally set to etcd when the kvdb_etcd build tag is
 // defined, allowing testing our database code with etcd backend.
-const TestBackend = EtcdBackendName
+const EtcdBackend = true
```

Comment:
Side question, but isn't it time to deprecate etcd for 0.16 because it can't move along to the sql world?

Reply:
I think we can start discussing that in the PR that makes the SQL based invoice DB the default, as only that will not be compatible with `etcd` anymore. This PR isn't the right place.